### DISCUSSION TITLE
Create account in supabase without logging out current user. Is it possible?

### DISCUSSION THREAD

**Discussion by wrteam-priyansh** [supabase/supabase#5467] [General]
Here is my scenario.
I have study management flutter app. Where teacher creates the accounts of student and then student can login in app. 
Teacher login in app and create student account but it logout teacher and signIn in with newly created student account. What i want is keep the teacher logged in and also create new student account. Can i do like this?

Hope i make sense

**Comment by guizoxxv** [comment]
I have the same issue. I think that the `signUp` function should have a boolean parameter `shouldSignIn` (default `true`) to control this behavior.

**Comment by Mikilll94** [comment]
I also have the same issue. I just want to create an account for another user.

**Reply by ioucyf** [reply]
@Mikilll94 have you checked the solution I suggested?

**Reply by MrMoussie** [reply]
You have a solution to this problem?

**Reply by hansalves** [reply]
@ofeenee where did you suggest this solution?
